wire-ios-notification-engine/WireNotificationEngine.xcodeproj/project.pbxproj

```diff
@@ -611,7 +611,7 @@
 					"@executable_path/Frameworks",
 					"@loader_path/Frameworks",
 				);
-				ONLY_ACTIVE_ARCH = NO;
+				ONLY_ACTIVE_ARCH = YES;
```

Comment:
Why did this change?

Reply:
This is because the core crypto library doesn't support `arm64`. In `wire-ios` this setting was already the default, allowing us to build with core crypto on intel Macs (because `arm64` is not an active architecture there).
Now, since core crypto was moved to data model, we need to replicate this setting from the leaf to the root node of the dependency tree.

Unfortunately, this means it won't build for M1 Macs when building for simulator (which is already the case?). 
This is also what was blocking us from using remote swift package earlier. Because core crypto team couldn't build an xcframework from the library. It's possible they have a fix already, we'll need to ask them

Reply:
>  this means it won't build for M1 Macs when building for simulator

Right... this is the problem I was facing. So CoreCrypto is blocking us all from building for simulator in Apple silicon. This will be a problem for CI machines running Apple silicon. For now we can accept it but we absolutely need to fix this.

Reply:
Completely agree. I talked to cc team and they can build for m1 simulator now so they'll do a release soon, that way we'll be able to use the remote package